src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -331,6 +338,21 @@
   }
 }
 
+// Toggle image
+.pf-c-dropdown__toggle-image {
+  display: inline-flex;
+  margin-top: var(--pf-c-dropdown__toggle-image--MarginTop);
+  margin-bottom: var(--pf-c-dropdown__toggle-image--MarginBottom);
+
+  &:not(:first-child) {
```

Comment:
could you add margin-left to above, and then in the selector `&:first-child` remove the margin left?

Reply:
After looking at the placement, if `.pf-c-dropdown__toggle-image` precedes `.pf-c-dropdown__toggle-text`, `margin-right` works well. If if follows `.pf-c-dropdown__toggle-icon`, `.pf-c-dropdown__toggle-image` doesn't need `margin-left`.